cluster/addons/metrics-server/resource-reader.yaml

```diff
@@ -6,31 +6,19 @@ metadata:
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  - nodes
-  - nodes/stats
-  - namespaces
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - "apps"
-  resources:
-  - deployments
-  verbs:
-  - get
-  - list
-  - update
-  - watch
-  - patch
-- nonResourceURLs:
```

Comment:
We dropped /metrics which is needed

https://github.com/kubernetes/kubernetes/pull/118122 adds it back

Reply:
Also deployments